test/AElf.Contracts.Parliament.Tests/ParliamentContractTest.cs

```diff
@@ -753,7 +781,7 @@ private async Task<Hash> CreateProposalAsync(ECKeyPair proposalKeyPair, Address
                 Symbol = "ELF",
                 Amount = 100,
                 To = Tester,
-                Memo = "Transfer"
+                Memo = Guid.NewGuid().ToString()

```

Comment:
It's better not use the random string.

Reply:
> It's better not use the random string.

Add random string just for different proposal tx, have been add comments for it.